src/Distribution/Nixpkgs/Haskell/FromCabal/PostProcess.hs

```diff
@@ -129,7 +129,7 @@ hooks =
   , ("hoogle", set testTarget "--test-option=--no-net")
   , ("hsignal < 0.2.7.4", set phaseOverrides "prePatch = \"rm -v Setup.lhs\";") -- https://github.com/amcphail/hsignal/issues/1
   , ("hslua < 0.9.3", over (libraryDepends . system) (replace (pkg "lua") (pkg "lua5_1")))
-  , ("hslua >= 0.9.3 && < 2.0.0", over (libraryDepends . system) (replace (pkg "lua") (pkg "lua5_3")))
+  , ("hslua >= 0.9.3 && < 2.2.0", over (libraryDepends . system) (replace (pkg "lua") (pkg "lua5_3")))
```

Comment:
Looking at the cabal file for `hslua-2.0.0` and `hslua-2.1.0`, it looks like there are no system packages needed.  Is this change here correct?

For example: https://hackage.haskell.org/package/hslua-2.0.0/hslua.cabal